api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -46,6 +47,10 @@
 
   private static final Logger logger = Logger.getLogger(FreeTierBillingService.class.getName());
 
+  // somewhat arbitrary
+  private static final double COST_COMPARISON_TOLERANCE = 0.0001;
+  private static final double COST_FRACTION_TOLERANCE = 0.0001;
```

Comment:
There's probably a way to figure this out based on use cases. I know from geometry angle and linear tolerance are different too. This is probably OK.